uxarray/grid.py

```diff
@@ -137,24 +127,15 @@ def __from_vert__(self):
             })
 
     # load mesh from a file
-    def __from_file__(self):
-        """Loads a mesh file Also, called by __init__ routine This routine will
-        automatically detect if it is a UGrid, SCRIP, Exodus, or shape file.
-
-        Raises:
-
-            RuntimeError: Unknown file format
-        """
-        # call function to set mesh file type: self.mesh_filetype
-        self.mesh_filetype = determine_file_type(self.filepath)
-
+    def __from_ds__(self):
```

Comment:
Could you please reflect this function renaming in documentation, docs/internal_api/index.rst?